harness/determined/cli/notebook.py

```diff
@@ -13,13 +15,48 @@
 from determined.common.declarative_argparse import Arg, Cmd, Group
 
 
+def expected_exceptions(func: Callable[[argparse.Namespace], Any]) -> Callable[..., Any]:
+    def wrapper(args: argparse.Namespace) -> Any:
+        try:
+            return func(args)
+        except Exception as e:
+            """
+            DISCUSS: is this a good pattern in Python? if so we could do this at argparse.Cmd
```

Comment:
thoughts?

Reply:
I separated this into a different branch https://github.com/determined-ai/determined/compare/master...hamidzr:determined:cli-err-handling?expand=1 and I'll follow up on it separately. switched this to a more local solution